pages-example-forum-app/worker-durable/wrangler.toml

```diff
@@ -0,0 +1,12 @@
+compatibility_date = "2022-07-21"
+main = "src/index.ts"
+name = "worker-durable"
+
+[durable_objects]
+bindings = [
+  {name = "Learning_DurableObjects", class_name = "DurableObject"},
```

Comment:
I would personally stick with a single naming (either camel or snake case)

Reply:
Any other reason for this so I can be mindful in the future?